087-single-step-multi-version-downgrade.md

```diff
@@ -0,0 +1,81 @@
+# Support Single step multi version downgrade for KRaft based clusters 
+
+This proposal seeks to introduce support for multi-version downgrade of Strimzi in a single step where possible.
+The proposal only relates to KRaft based clusters.
+
+## Current situation
+
+- Downgrading of Strimzi is currently supported where the Kafka version in use is supported by the 'to' Strimzi version. 
+- Downgrading of Strimzi is not supported if the Kafka version in use is unsupported by the 'to' Strimzi version. Attempting this will result in an error during reconciliation of the Kafka CR. Downgrading the Kafka version to a version supported by the 'to' Strimi version will not resolve the reconciliation failure as the 'from' Kafka version is unknown to the 'to' Strimzi version (in contrast to the upgrade scenario where upgrading the Kafka version will enable reconciliation to subsequently succeed). The only way to successfully execute such a downgrade path is to iteratively downgrade Kafka and Strimzi ensuring at each step that the current Kafka version supported by the Strimzi version.  
+- Downgrading of Kafka where the metadata version in use is greater than the maximum supported by the 'to' Kafka version is not supported
+
+## Motivation
+
+This proposal would make downgrade of Strimzi more user friendly, specifically, in the scenario where a single-step multi version upgrade has been performed and a need arises to downgrade before the metadata version has been advanced. The proposal will allow the user perform the roll back in a single step rather than the multiple steps currently required.
```

Comment:
```suggestion
This proposal would make downgrade of Strimzi more user friendly, specifically, in the scenario where a single-step multi version upgrade has been performed and a need arises to downgrade before the metadata version has been advanced. The proposal will allow the user perform the downgrade in a single step rather than the multiple steps currently required.
```